### DISCUSSION TITLE
Linking of manual instrumentation and automatic instrumentation traces

### DISCUSSION THREAD

**Discussion by Rishibha-tech** [open-telemetry/opentelemetry-java#3021] [Q&A]
How do we link manual instrumentation traces with automatic instrumentation traces in open telemetry . They are not coming under same hierarchy

**Comment by iNikem** [comment]
Please make sure that you use compatible versions of instrumentation javaagent and Otel Java API. [Each release of instrumentations](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases) specifies which Otel Java API version it is compatible with.

This strict version matching will be somewhat relaxed in the future :)
